<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>resource_packageClass</class>
 <widget class="QWidget" name="resource_packageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>resource_package</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>160</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label1_Device_Access_Protocol">
      <property name="text">
       <string>设备接入协议</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label2_Data_Access_Protocol">
      <property name="text">
       <string>数据接入协议</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label3_Manufacturer_Name_CN">
      <property name="text">
       <string>厂商中文名称</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label4_Manufacturer_Name_US">
      <property name="text">
       <string>厂商英文名称</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>160</y>
     <width>160</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="lineEdit1_Device_Access_Protocol"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit2_Data_Access_Protocol"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit3_Manufacturer_Name_CN"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit4_Manufacturer_Name_US"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton1_Cancel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton3_Confrim">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton2_Reset">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource_package.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton1_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>resource_packageClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
